### PR TITLE
News feature

### PR DESCRIPTION
**Pull request by RobLoweMOJ**

### CODE REVIEW

File: archive-news.php [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        </h1>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-third">
+                <div class="news-archive-filter-section">
+                    <p>You can use the filters to show only news items that match your interests.</p>
+
+                    <div class="news-archive-filter-form">
+                        <form method="GET">
+
+                            <div></div>
```
Comment (RobLoweMOJ) [inline archive-news.php:31]: Remove

File: archive-news.php [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+                </div>
+            </div>
+            <div class="govuk-grid-column-two-thirds">
+                <?php
+                if (have_posts()) {
+                    $news_story_count = $GLOBALS['wp_query']->found_posts;
+                    ?>
+                    <div class="news-story-count">
+                        <?php echo $news_story_count; ?> articles
```
Comment (RobLoweMOJ) [inline archive-news.php:97]: Add logic for singular

File: assets/js/news-archive.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+jQuery( document ).ready(function( $ ) {
+
+  $('#news-archive-filter-topic').on('change', function() {
+
+    resetSubTopics();
+
+    var selected_topic = this.value;
+
```
Comment (RobLoweMOJ) [inline assets/js/news-archive.js:8]: numeric check

File: assets/scss/news.scss [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+  }
+
+  .news-story-featured-image {
+    margin-top: 25px;
+  }
+}
+
+.archive,
+.post-type-archive-news {
```
Comment (RobLoweMOJ) [inline assets/scss/news.scss:29]: chop it OUT!

File: assets/scss/news.scss [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+  .news-archive-filter-section {
+    p {
+      margin-top: 0;
+    }
+
+    .news-archive-filter-form {
+
+
```
Comment (RobLoweMOJ) [inline assets/scss/news.scss:38]: chop space

File: assets/scss/news.scss [hunk 6]
```diff
@@ -0,0 +1,105 @@
+.single-news {
```
Comment (RobLoweMOJ) [inline assets/scss/news.scss:1]: chop

File: assets/scss/news.scss [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+  }
+
+  .news-story-list-item {
+    border-bottom: 1px solid $colour_concrete_grey;
+
+    &:last-child {
+      border: none;
+    }
+
+    .news-story-list-item-title {
```
Comment (RobLoweMOJ) [inline assets/scss/news.scss:68]: Branding

File: assets/scss/news.scss [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+  .news-story-categories-list {
+    list-style: none;
+    padding: 0;
+    margin-top: 0;
+
+    .news-story-categories-list-item {
+      display: inline-block;
+      margin: 24px 0 0 0;
+
+      .news-story-category-link {
```
Comment (RobLoweMOJ) [inline assets/scss/news.scss:93]: hover/focus black with white text

File: assets/scss/style.scss [hunk 9]
```diff
@@ -33,3 +33,6 @@ $gutenberg-styles: false;
 //icons for footer
 @import 'icons';
 @import 'condolence-banner';
```
Comment (RobLoweMOJ) [inline assets/scss/style.scss:35]: chop

File: category.php [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+            echo 'Topic: ' . single_cat_title('', false);
+            ?>
+        </h1>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <?php
+                if (have_posts()) {
+                    while (have_posts()) {
+                        the_post(); ?>
+                        <div class="news-story-list-item">
```
Comment (RobLoweMOJ) [inline category.php:30]: template

File: template-parts/content-news.php [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+        <div class="news-story-excerpt intro">
+            <?php the_excerpt(); ?>
+        </div>
+
+    <?php
+    }
+    ?>
+	<?php do_action( 'hale_before_single_content' ); ?>
+
+	<div class="">
```
Comment (RobLoweMOJ) [inline template-parts/content-news.php:84]: Give class name

**Review by MalcolmVonMoJ — APPROVED** [review]

**Pull request merged**